api/webauth/templates/merge.html

```
@@ -0,0 +1,57 @@
+{{template "base_start" .}}
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)